tmail-backend/jmap/extensions/src/main/java/com/linagora/tmail/james/jmap/AttendanceStatus.java

```diff
@@ -0,0 +1,5 @@
+package com.linagora.tmail.james.jmap;
+
+public enum AttendanceStatus {
+    Accepted, Tentative, Delegated, NeedsAction, Declined
```

Comment:
what do you need the `Delegated` status for ? (not used anywhere)

Reply:
Delegated is declared in the [specification](https://datatracker.ietf.org/doc/html/rfc5545#section-3.2.12), but it is not in use for now, neither in the frontend nor the backend.